src/components/BottomSheet.tsx

```diff
@@ -0,0 +1,118 @@
+import styled from 'styled-components';
+import { IcCancel } from '../assets/icons';
+import { FONT_STYLES } from '../styles/fonts/font';
+import { BtClose } from '../assets/buttons';
+import { useNavigate } from 'react-router-dom';
+
+export default function BottomSheet({ onToggleModal }: any) {
```

Comment:
```suggestion
interface BottomSheetProps{
  onToggleModal : () => void;
}
export default function BottomSheet({ onToggleModal }: BottomSheetProps) {
```
저도 공부한 지 얼마 안 돼서 잘 모르겠지만 interface나 type으로 props 형태를 지정해주면 되려나? @NamJwong  도와주세요 ㅎㅎ!!

Reply:
저렇게 하면 됩니다!!!!!!!!!!! 굳굳굳

Reply:
아하 좋습니다!! 👍